modules/tensor_mechanics/src/materials/ADPowerLawCreepStressUpdate.C

```diff
@@ -78,3 +78,18 @@ ADPowerLawCreepStressUpdate::computeDerivative(const ADReal & effective_trial_st
                                        _exp_time;
   return creep_rate_derivative * _dt - 1.0;
 }
+
+void
+ADPowerLawCreepStressUpdate::computeStrainEnergyRateDensity(
+    ADMaterialProperty<Real> & strain_energy_rate_density,
+    const ADMaterialProperty<RankTwoTensor> & stress,
+    const ADMaterialProperty<RankTwoTensor> & strain_rate)
+{
+  if (_n_exponent < 1)
```

Comment:
Does the exponent always need to be >1? In that case, maybe it should be a range-checked parameter.

Reply:
There is one test in `solid_mechanics` where the exponent is identically one: `./solid_mechanics/test/tests/combined_creep_plasticity/plasticity_only_combined_class_sm1.i`

But probably, for power law creep, it should be strictly greater than one.

Reply:
There is one test with n exponent identically one: `./solid_mechanics/test/tests/combined_creep_plasticity/plasticity_only_combined_class_sm1.i`

But it looks like power law creep requires exponent greater than one. Here, the intention should be avoiding division by zero. If the range is added in the material it would prevent the user from using it if it wants to experiment with exponent unity.